stable/pgadmin/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: pgAdmin is a web based administration tool for PostgreSQL database
 name: pgadmin
-version: 1.2.0
+version: 1.2.1
```

Comment:
Can you bump this towards 1.2.2, since https://github.com/helm/charts/pull/20804 is still not merged and that will use 1.2.1. Apologies, I am depended on other reviewers.

Reply:
done